.github/workflows/tests.yml

```
@@ -111,16 +120,16 @@
       run: dotnet restore ${{ env.SOLUTION_PATH }}
       
     - name: Build solution
-      run: dotnet build ${{ env.SOLUTION_PATH }} --configuration ${{ matrix.configuration }} --no-restore
+      run: dotnet build ${{ env.SOLUTION_PATH }} --configuration Test --no-restore
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)